docs/snippets/ov_extensions.py

```diff
@@ -38,3 +38,18 @@ def conversion(node):
 
 core.add_extension(ConversionExtension("ThresholdedRelu", conversion))
 #! [py_frontend_extension_ThresholdedReLU]
+
+
+#! [py_frontend_extension_aten_hardtanh]
+from openvino.frontend import FrontEndManager, ConversionExtension, NodeContext
+
+def convert_hardtanh(node: NodeContext):
+    inp = node.get_input(0)
+    min_value = node.get_values_from_const_input(1)
+    max_value = node.get_values_from_const_input(2)
+    return ops.clamp(inp, min_value, max_value).outputs()
+
+fem = FrontEndManager()
+fe = fem.load_by_framework(framework="pytorch")
+fe.add_extension(ConversionExtension("aten::hardtanh", convert_hardtanh))
```

Comment:
I think this is strange use case for conversion. I propose to register conversion extensions also via core.
Frontend API is more developer API and not public one.
What do you think?

Reply:
This extension could be registered via core for other frontends, but for pytorch we have to use Frontend::add_extension directly.

Reply:
`read_model` is not suitable to register an extension. `convert_model` is only recommended way to register an extension for PT FE.
@slyalin, cc

Reply:
@rkazants I changed the example to use convert_model. Can you take a look now?